config/locales/views/pupils/pupils.yml

```diff
@@ -0,0 +1,67 @@
+---
+en:
+  pupils:
+    analysis:
+      compare_electricity_in_2_weeks: compare the electricity used in two different weeks
+      compare_electricity_on_2_days: compare the electricity used on two different days
+      compare_electricity_use_by_meters: compare the electricity use by different meters
+      compare_gas_in_2_weeks: compare the gas used in two different weeks
+      compare_gas_on_2_days: compare the gas used on two different days
+      compare_gas_use_by_meters: compare the gas use by different meters
+      electricity_and_solar_data_html: Here's the <strong>electricity and solar</strong> data for your school
+      electricity_data_html: Here's the <strong>electricity</strong> data for your school
+      explore_data: Explore data
+      explore_energy_data_html: Explore the <strong>%{fuel_type}</strong> data for your school
+      find_electricity_comparison: find out how my school's electricity use compares with other schools
+      find_electricity_use_7days: find out how much electricity was used in the last 7 days
+      find_electricity_use_last_year: find out how much electricity was used last year
+      find_electricity_use_lights: find out how much electricity was used by lights and appliances running all the time
+      find_electricity_use_long_term: find out how electricity use has changed long term
+      find_gas_comparison: find out how my school's gas use compares with other schools
+      find_gas_use_7days: find out how much gas was used in the last 7 days
+      find_gas_use_last_year: find out how much gas was used last year
+      find_gas_use_long_term: find out how gas use has changed long term
+      find_how_much_co2_generated_html: find out how much <strong>carbon dioxide</strong> was generated
+      find_how_much_electricity_cost_html: find out <strong>how much</strong> the electricity cost
+      find_how_much_gas_cost_html: find out <strong>how much</strong> the gas cost
+      find_how_much_storage_heaters_cost_html: find out <strong>how much</strong> the storage heaters cost
+      find_out_about_solar_html: find out about <strong>solar</strong>
+      find_storage_heaters_comparison: find out how my school's storage heater use compares with other schools
+      find_storage_heaters_use_last_year: find out how much electricity was used by storage heaters last year
+      find_storage_heaters_use_long_term: find out how storage heater electricity use has changed long term
+      find_when_electricity_used_html: find out <strong>when</strong> the electricity was used
+      find_when_gas_used_html: find out <strong>when</strong> the gas was used
+      find_when_storage_heaters_used_html: find out <strong>when</strong> the storage heaters used electricity
+      gas_data_html: Here's the <strong>gas</strong> data for your school
+      i_want_to: I want to&hellip;
```

Comment:
I'm wondering whether we need to use the HTML entity here rather than just "…".

If we switched to using the UTF-8 symbol then we can remove calling sanitize on this when included. https://techoverflow.net/2019/08/05/what-is-hellip-in-html/